swan-lake/learn-the-platform/source-code-dependencies/manage-dependencies.md

```diff
@@ -208,3 +208,38 @@ error: compilation failed: Two incompatible versions exist in the dependency gra
 ballerina/observe versions: 0.9.0, 1.0.0
 ```
 
+## Manage Platform Dependencies
+
+A Ballerina package can depend on JAVA code that are shipped with the JVM, from a remote package repository or from a JAR file located in the user’s machine. Libraries shipped with the JVM can be used seamlessly and libraries used from other locations must be specified in the `Ballerina.toml` as shown below.
+
+## Specify a Maven dependency
+
+```toml
+[[platform.java11.dependency]]
+# Group ID of the Maven dependency.
+groupId = "<group-id>"
+# Artifact ID of the Maven dependency.
+artifactId = "<artifact-id>"
+# Version of the Maven dependency.
+version = "<version>"
+```
+
+When building the package, these specified Maven dependencies will be resolved and can be found in the `target/platform-libs` directory. 
+
+### Specify a local JAR file path
```

Comment:
```suggestion
### Specify a local JAR file path
```
Can we add a one-line intro description after this topic?

Reply:
Added in https://github.com/ballerina-platform/ballerina-dev-website/commit/b6a5ef1ab01b17dadaa434eb584fa1a665109a08